Assets/BezierCurve.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BezierCurve : MonoBehaviour
+{
+    // The task asks to implement quadratic Bezier Curve.
+    // However, there is support for higher order Bezier Curves.
```

Comment:
It is good that you learned how to implement them. However, they are not useful because of their computational inefficiency (because of all those factorials and combinations).